scripts/spidhub-fargate/environments/dev/storage/config/spid-saml-check/v1/configvolume/spid_users.json

```diff
@@ -0,0 +1,562 @@
+[
+	{
+		"username": "cesare",
+		"password": "password123",
+		"spidLevel": 1,
+
+		"spidCode": "SPID-001",
+		"name": "Gaio Giulio",
+		"familyName": "Cesare",
+		"gender": "M",
+		"dateOfBirth": "1944-07-13",
+		"countyOfBirth": "RM",
+		"placeOfBirth": "H501",
+		"idCard": "cartaIdentita AC137100 comuneRoma 2007-12-22 2018-07-13",
+		"expirationDate": "2019-04-11",
+		"fiscalNumber": "TINIT-CSRGGL44L13H501E",
+		"mobilePhone": "3331234567",
+		"email": "gaiogiuliocesare@gmail.com",
+		"digitalAddress": "",
+		"ivaCode": "",
+		"companyName": "",
+		"registeredOffice": "",
+		"companyFiscalNumber": "",
+		"address": "Via Listz 21 00144 Roma",
+		"domicileStreetAddress": "Via Listz 21",
+		"domicilePostalCode": "00144",
+		"domicileMunicipality": "Roma",
+		"domicileProvince": "RM",
+		"domicileNation": "IT"
+	},
+	{
+		"username": "ada",
+		"password": "password123",
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

It appears your code contains a hardcoded password. Hardcoded secrets or credentials can allow attackers to bypass authentication methods and perform malicious actions. We recommend revoking access to resources using this credential and storing future credentials in a management service such as [AWS Secrets Manager](https://aws.amazon.com/secrets-manager/).

[Learn more about the use of hardcoded credentials](https://cwe.mitre.org/data/definitions/798.html)